indra/llwindow/llkeyboardsdl.cpp

```diff
@@ -24,7 +24,7 @@
  * $/LicenseInfo$
  */
 
-#if LL_SDL
+#if LL_LINUX
```

Comment:
It used to be the case only the linux build used **SDL**, but going forward with **SDL2** it is used for ALL platforms (well... except for DARWIN which restricts game controllers at the OS level), so I'm replacing `LL_SDL `with `LL_LINUX`.